languages/julia/exercises/concept/fibonacci-iterator/.docs/instructions.md

```diff
@@ -0,0 +1,87 @@
+In this exercise you're going to write an iterator that iterates through the first `n` elements of the [Fibonacci sequence](https://en.wikipedia.org/wiki/Fibonacci_number).
+
+## Fibonacci sequence
+
+The Fibonacci sequence is a sequence of numbers, where every element of the sequence is calculated by adding the two numbers that come before it.
+You can think of a sequence as a list of numbers in a certain order.
+The first two numbers of the sequence are set to $1$.
+
+The start of the sequence is $1,~1,~2,~3,~5,~8,~13,~21,~34,~55,~89,~\dots$
+
+In maths notation: <!-- TODO: Move this to an info box once they're available -->
+
+$$
+
+a_1 = a_2 = 1\\
+a_n = a_{n-1} + a_{n-2}
+
+$$
```

Comment:
Does this render correctly on our website? GitHub doesn't seem to render it

Reply:
Eventually it will: https://github.com/exercism/exercism/issues/5277

Reply:
Awesome! Thanks